serve-api/repositories/order-repo.go

```diff
@@ -0,0 +1,47 @@
+package repositories
+
+import (
+	"github.com/ministryofjustice/serve-opg/serve-api/entity"
+	"gorm.io/gorm"
+	"time"
+)
+
+type OrderRepo struct {
+	db *gorm.DB
+}
+
+func NewOrderRepo(db *gorm.DB) *OrderRepo {
+	return &OrderRepo{
+		db: db,
+	}
+}
+
+// GetServedOrders will get all orders from the DB
+func (r *OrderRepo) GetServedOrders(dateLimit ...time.Time) ([]entity.Order, error) {
```

Comment:
The live service report now gets orders served within the last 4 weeks, however I wanted to allow this filter to be an optional param (optional params arent a thing in go, so from google this is what most results recommend as a way around it)

Reply:
I guess my only comment would be do we want to add some constraint to stop users from entering multiple dates here?